src/components/PlanningTable/Columns/Title.tsx

```diff
@@ -0,0 +1,38 @@
+import { type ColumnDef } from '@tanstack/react-table'
+import { TooltipProvider, Tooltip, TooltipTrigger, TooltipContent } from '@ttab/elephant-ui'
+import { cn } from '@ttab/elephant-ui/utils'
+import { type Planning } from '../data/schema'
+
+export const title: ColumnDef<Planning> = {
+  id: 'title',
+  accessorFn: (data) => data._source['document.title'][0],
+  cell: ({ row }) => {
+    const status = row.original._source['document.meta.core_planning_item.data.public'][0] === 'true'
+    const slugline = row.original._source['document.meta.core_assignment.meta.tt_slugline.value']
+
+    return (
+        <div className='flex space-x-2 w-fit'>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger>
+                <div className={cn(
+                  'flex items-center h-2 w-2 rounded-full mx-4',
+                  status
+                    ? 'bg-[#5895FF]'
+                    : 'border')} />
+              </TooltipTrigger>
+              <TooltipContent>
+                <p>{status ? 'Public' : 'Internal'}</p>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+          <span className='max-w-[200px] md:max-w-[300px] lg:max-w-[700px] truncate font-medium'>
+            {row.getValue('title')}
+          </span>
+          {slugline?.length && (
```

Comment:
This type of construct will happily render a "0" as React does not turn it into a boolean. It´s safer to always make sure it is a boolean like
`{!!slugline?.length && ( ...)}`